services/apps/alcs/src/providers/typeorm/migrations/1707851589615-add_advanced_parcel_mapping.ts

```diff
@@ -0,0 +1,33 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddAdvancedParcelMapping1707851589615
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+    UPDATE
```

Comment:
I do not think you can set false to everything since we have actual users submitting applications now. This should only apply to etl data